flutter_local_notifications/example/android/app/src/main/AndroidManifest.xml

```diff
@@ -39,5 +39,12 @@
         <meta-data
             android:name="flutterEmbedding"
             android:value="2" />
+        <service
+            android:name="com.dexterous.flutterlocalnotifications.ForegroundService"
+            android:enabled="true"            
+            android:exported="false"
+            android:stopWithTask="true"
+            android:foregroundServiceType="microphone" />
```

Comment:
Yes, it will work even if this particular entry is removed, I just wanted to give an example how to use `android:foregroundServiceType`.
It seems that this intention was not clear... We could either remove it, or annotate it with a comment like:
```xml
            android:enabled="true"            
            android:exported="false"
            android:stopWithTask="true"
	    <!-- "microphone" is an example, you can omit the foregroundServiceType tag entirely.
            For suitable values see https://developer.android.com/guide/topics/manifest/service-element  -->
            android:foregroundServiceType="microphone" />
```
What do you think?

Reply:
As it's realistically not needed then removing it would be appropriate so it's more realistic. Even with a comment, some  developers may not look close enough. This has happened before like how there's a comment in the example app stating that just copying the Dart code over won't work but it happens and then issues are raised that things aren't working